docs/docs-site/content/user/concept/_index.md

```diff
@@ -66,6 +66,14 @@ Shared documents will show-up with a little blue icon.
 
 ![Sharing](https://cdn.gethue.com/uploads/2019/04/editor_sharing.png)
 
+Along with document sharing, there is another query sharing option via links which is quicker and easier - ![Public Links/Gist Sharing](https://gethue.com/blog/2020-03-04-datawarehouse-database-sql-collaboration-and-sharing-with-link-and-gist/)
```

Comment:
Less generic and content info? (i.e. moving content from the post here and not pointing to the blog post)

- Public link: quickly sharing parameterized saved reports, customer analyses links
- Gist: quickly share SQL snippets, with a direct link to the SQL editor

... etc

and images?

Reply:
Added the content from blog post